include/depthai/depthai_wrapper.hpp

```diff
@@ -0,0 +1,57 @@
+#include "depthai/device.hpp"
+#include <string>
+#include <opencv2/opencv.hpp>
+#include <opencv2/core/core.hpp>
+#include <fstream>
+#include <sstream>
+#include "nlohmann/json.hpp"
+#include <vector>
+
+namespace DepthAI
+{
+
+using CV_mat_ptr = std::shared_ptr<cv::Mat>;  // shared ptr for cv::Mat
+
+class DepthAI : public Device
```

Comment:
I am not sure if it's a good idea to call this library DepthAI. It's too generic and doesn't tell me anything about what it does and how it interfaces with OAK-D. It is also the name of the Luxonis product which makes it confusing. Why don't you call it `OAKWrapper` or something more descriptive?

Reply:
Also, the name of the file should match with the name of the class it implements. So you'll need to change that as well.

Reply:
Depthai is their library name used to connect to the devices OAK and OAK-D are just two devices which uses it.